EnglishFilter/sections/adservers.txt

```diff
@@ -5,6 +5,7 @@
 !
 !! https://a.bestcontentitem.top/warp/*?r=
 !
+||feyl4mh5to.com^
```

Comment:
Why not `||feyl4mh5to.com^$third-party`?
Unfortunately, I don't see it on my end with US IP. Can you provide a screenshot?

Reply:
-_- sure

Reply:
@zloyden
This request blocks by regesp, check here https://2shrt.com/n6xQG

Reply:
@zloyden It's obsolete now! Let me remove  it.

Reply:
Ok, I found it and agree with this rule.

Reply:
Why? @zloyden. It's annoying when one do not check properly and comment. @zloyden Please.

Reply:
@wout-codes
Please turn the rule back, f.e. iOS doesn't support regexps so this rule will be useful

Reply:
I did.